src/scenegraph/Parser.cpp

```diff
@@ -180,6 +180,14 @@ namespace SceneGraph {
 					throw ParseError("Animation start/end frames seem wrong");
 				m_model->animDefs.push_back(AnimDefinition(animName, startFrame, endFrame, loopMode));
 				return true;
+			} else if (match(token, "bound_thick_line")) {
```

Comment:
Personally, I think `bound_cylinder` is a better choice of name here, but I'll leave it to your discretion whether thick_line is the technically correct name.

Reply:
I've chosen "capsule" as a name instead, as this matches the typical capsule collider used in most game engines (cylinder with rounded endcaps)

Reply:
Definitely a better choice - I would not have been able to tell that was the underlying shape from the existing nomenclature.